pyproject.toml

```diff
@@ -33,3 +33,82 @@ setuptools = ">=69.5.1,<75.0.0"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
+# Exclude a variety of commonly ignored directories.
+exclude = [
```

Comment:
We should probably add some contentctl artifacts here too:
* `downloads/`
* `test_results/`

Reply:
Do these end up _in_ the contentctl repo, or just in repos created with the tool?

Reply:
Ahh good point, this comment can be disregarded. I was thinking from the context of `security_content`